drivers/ipm/Kconfig

```diff
@@ -75,3 +75,21 @@ config IPM_IMX_MAX_ID_VAL
 	default 1 if IPM_IMX_MAX_DATA_SIZE_8
 	default 0 if IPM_IMX_MAX_DATA_SIZE_16
 	depends on IPM_IMX
+
+config IPM_MHU
+	bool "IPM MHU driver"
+	depends on IPM
+	help
+	  Driver for SSE 200 MHU (Message Handling Unit)
+
+config IPM_MHU0
```

Comment:
Do we need instance defines, can we get this from DT?

`DT_ARM_MHU_0` or `DT_ARM_MHU_1`

Reply:
DT_ARM_MHU_0 and DT_ARM_MHU_1 are always defined after mhu added into dts file, are they?
The config here provide a choice which MHU device should add as in this sub system include 2 MHU.
When enabling MHU device, there will be settings in the prj.conf like:

CONFIG_IPM=y
CONFIG_IPM_MHU=y
CONFIG_IPM_MHU0=y
CONFIG_IPM_MHU1=y

Reply:
But that choice of which MHU can and should be handled by whats in the DTS.   I don't think we need the Kconfig options here.